models/mx_sesnsp_incidencia_delictiva/code/upload.py

```
@@ -0,0 +1,111 @@
+
+
+def upload(slug):
+    path = OUT / slug
+    if not path.exists():
+        raise FileNotFoundError(path)
+    expected = local_rows(slug)
+    tb = bd.Table(dataset_id=DATASET_ID, table_id=slug)
+    st = bd.Storage(dataset_id=DATASET_ID, table_id=slug)
+    try:
+        st.delete_table(mode="staging", not_found_ok=True)
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)